lib/motion/test_helpers.rb

```diff
@@ -16,14 +16,18 @@ def motion?(component, motion_name)
       component.motions.include?(motion_name.to_s)
     end
 
-    def run_motion(component, motion_name)
+    def run_motion(component, motion_name, event = motion_event)
       if block_given?
         c = component.dup
-        c.process_motion(motion_name.to_s)
+        c.process_motion(motion_name.to_s, event)
         yield c
       else
-        component.process_motion(motion_name.to_s)
+        component.process_motion(motion_name.to_s, event)
       end
     end
+
+    def motion_event(attributes = {})
+      Motion::Event.new(ActiveSupport::JSON.decode(attributes.to_json))
```

Comment:
Round-trip to json and back so we can accept hashes with symbols and to better mimic how an event would end up in a motion. Event and Element expect string-based hashes and events from the browser will come in with string keys.

Reply:
You can also do `attributes.deep_stringify_keys`

```ruby
pry(main)> { foo: { bar: 1 }}.deep_stringify_keys
=> {"foo"=>{"bar"=>1 } }
```